### PR TITLE
Fix choix date in survey

### PR DESCRIPTION
**Pull request by FHenry**

### CODE REVIEW

**Comment by eldy** [comment]
You must check on both var:
varname and varname_x to be compatible with all browsers. Also there is a print 'toto' to remove.

File: htdocs/opensurvey/wizard/choix_date.php [hunk 1]
```diff
@@ -251,13 +251,14 @@
...
 //mise a jour des valeurs de session si mois apres
-if (issetAndNoEmpty('moisapres')) {
+if (issetAndNoEmpty('moisapres_x')) {
 	if ($_SESSION["mois"] == 12) {
 		$_SESSION["mois"] = 1;
 		$_SESSION["annee"] += 1;
 	} else {
 		$_SESSION["mois"] += 1;
 	}
+	print 'toto';
```
Comment (rdoursenaud) [inline htdocs/opensurvey/wizard/choix_date.php]: lol, indeed

**Comment by eldy** [comment]
Are you sure we must close this ?

What about the bug ?
if (issetAndNoEmpty('moisavant')) {
that was fixed by 
f (issetAndNoEmpty('moisavant_x')) {

and that i think should be fixed with
if (issetAndNoEmpty('moisavant') || issetAndNoEmpty('moisavant_x')) {

**Comment by FHenry** [comment]
Ok I was thinking you've done it. Ok I repoen and fix on a correct way

**Pull request merged**